tests/tests_input_file.yml

```diff
@@ -0,0 +1,28 @@
+- name: Ensure that the role runs with input file
+  hosts: all
+  gather_facts: false
+
+  tasks:
+    - name: Create table using sql script
+      block:
```

Comment:
You don't need a `block` here.  The cases where you do want to use `block`:
* you have some variables or settings you want to apply to a group of tasks - you can set `when` or `vars` at the `block` level rather than setting `when` or `vars` on each task - depending on the complexity, you can also move the related tasks to a separate tasks file - https://github.com/linux-system-roles/mssql/blob/master/tasks/main.yml#L17
* you have some clean up to do that you want to apply every time, even when there is an error e.g. you want to remove a temp file/dir, you want to restore some config file to its pre-test state - in this case, you use an `always` section to go with the `block` https://github.com/linux-system-roles/mssql/blob/master/tasks/main.yml#L782
* you want to "catch" an error and do something - in this case, you use `rescue` and do something with the error https://github.com/linux-system-roles/storage/blob/master/tests/tests_fatals_cache_volume.yml#L47